web_notify/tests/test_res_users.py

```diff
@@ -3,13 +3,13 @@
 
 import json
 
-import mock
-
 from odoo import exceptions
 from odoo.tests import common
 
 from odoo.addons.bus.models.bus import json_dump
 
+import mock
```

Comment:
We're on py3 now, and mock is in the stdlib:

```suggestion
from unittest import mock
```

Reply:
Ooops this is not related with the PR... pre-commit  changed it. I will discard this.

Reply:
No, pre-commit just changed the order of imports, but you really have to apply that change :grin:

Reply:
@Tardo I believe this is the only oustanding issue to solve before we can merge. Thanks!

Reply:
Ok, sorry... changed!